src/wp-admin/includes/nav-menu.php

```diff
@@ -566,15 +619,19 @@ function wp_nav_menu_item_post_type_meta_box( $object, $box ) {
 
 		<p class="button-controls wp-clearfix">
 			<span class="list-controls">
-				<a href="<?php
-					echo esc_url( add_query_arg(
-						array(
-							$post_type_name . '-tab' => 'all',
-							'selectall' => 1,
-						),
-						remove_query_arg( $removed_args )
-					));
-				?>#posttype-<?php echo $post_type_name; ?>" class="select-all aria-button-if-js"><?php _e( 'Select All' ); ?></a>
+				<a href="
+				<?php
```

Comment:
newline in `href`
I thought this was fixed, or was that a different nav-menu file?

Reply:
PR updated